mailpoet/lib/Subscription/Captcha.php

```diff
@@ -98,6 +99,33 @@ private function isUserExemptFromCaptcha() {
     return !empty(array_intersect($roles, (array)$user->roles));
   }
 
+  public function renderAudio($sessionId, $return = false) {
+
+    $audioPath = Env::$assetsPath . '/audio/';
```

Comment:
Feels like we can benefit from memoizing this function since it's working with IO, what do you think?